core/db/default/spree/stores.rb

```diff
@@ -5,5 +5,6 @@
     s.name              = 'Spree Demo Site'
     s.url               = 'demo.spreecommerce.com'
     s.mail_from_address = 'spree@example.com'
+    s.group             = 'surfdome'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.